src/main/java/com/minecolonies/apiimp/initializer/ModJobsInitializer.java

```diff
@@ -271,7 +271,7 @@ private ModJobsInitializer()
           .createJobEntry());
 
         ModJobs.cookassistant = register(DEFERRED_REGISTER, ModJobs.COOKASSISTANT_ID.getPath(), () -> new JobEntry.Builder()
```

Comment:
Is this done for backwards compat for the existing jobs to turn into chefs?
Could this not cause chefs to still be hired in the restaurant?

Reply:
I tested it locally and it worked fine. You can test it if you want. But we do have special code to fire jobs at wrong building.